installers/go-agent/win/config/wrapper-agent.conf

```diff
@@ -137,14 +133,17 @@ wrapper.ntservice.interactive=true
 #********************************************************************
 # Wrapper License Key
 #********************************************************************
-wrapper.license.type=DEV
-wrapper.license.id=200901090002
+#encoding=UTF-8
+wrapper.license.type=NODE
+wrapper.license.id=201610130000035
 wrapper.license.licensee=ThoughtWorks
-wrapper.license.dev_application=Cruise Agent
-wrapper.license.features=
-wrapper.license.upgrade_term.begin_date=2008-06-03
-wrapper.license.upgrade_term.end_date=2009-06-03
-wrapper.license.key.1=ae3e-a990-bff1-2fda
-wrapper.license.key.2=8657-f2d4-4fa2-9688
-wrapper.license.key.3=3c6e-1981-de9b-f0d6
-wrapper.license.key.4=c5f5-b5d4-bef8-0014
+wrapper.license.host_id=080027d3e5d0
+wrapper.license.features=pro, 64bit, trial
+wrapper.license.upgrade_term.begin_date=2016-10-13
+wrapper.license.upgrade_term.end_date=2016-11-13
+wrapper.license.lease_term.begin_date=2016-10-13
+wrapper.license.lease_term.end_date=2016-11-13
```

Comment:
This is a trial license that I have put up as is so we could test out the generated installers. The purchase of the actual one is in process, which might not have the dates hardcoded.